config/custom-environment-variables.json

```diff
@@ -149,7 +149,10 @@
     "clientId": "TRANSFERWISE_CLIENT_ID",
     "clientSecret": "TRANSFERWISE_CLIENT_SECRET",
     "redirectUri": "TRANSFERWISE_REDIRECT_URI",
-    "privateKey": "TRANSFERWISE_PRIVATE_KEY"
+    "privateKey": "TRANSFERWISE_PRIVATE_KEY",
```

Comment:
I agree with you, I'm just punting this for after I finish the OTT refactor because we have a lot more things to rename and I rather keep everything consistent.